boards/samd20-xpro/doc.txt

```diff
@@ -77,7 +80,7 @@ the SAMD20: http://watr.li/samr21-dev-setup-ubuntu.html
 ## Supported Toolchains
 
 For using the samd20-xpro board we strongly recommend the usage of the
-[GNU Tools for ARM Embedded Processors](https://launchpad.net/gcc-arm-embedded)
+[GNU Tools for ARM Embedded Processors](https://developer.arm.com/Tools%20and%20Software/GNU%20Toolchain)
 toolchain.
```

Comment:
I'd drop that, any `arm-none-eabi-gcc` is fine - I've been using the one from the Ubuntu repo for years.